.gitlab/corona-jobs.yml

```diff
@@ -30,7 +30,7 @@ corona_clang_hip:
 corona_clang_hip_tpls:
   parallel:
     matrix:
-      - COMPILER_SPEC: clang@15.0.0
+      - COMPILER_SPEC: rocmcc@5.4.1
```

Comment:
Why the change from `clang` to `rocmcc`?

Reply:
The radiuss-spack-configs renamed the clang compilers to their rocmcc version.